forge/src/main/java/org/openjdk/skara/forge/RefChange.java

```diff
@@ -0,0 +1,6 @@
+package org.openjdk.skara.forge;
+
+import java.time.ZonedDateTime;
+
+public record RefChange(String prevRefName, String curRefName, ZonedDateTime createdAt) {
```

Comment:
Again, I'm more for shorter names, so I would have gone with `ReferenceChange(String from, String to, ZonedDateTime at)`

Reply:
I like that.